### DISCUSSION TITLE
appendix_languages

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/labview_book#68] [Announcements]
# appendix_languages

这一节，记录笔者在使用 LabVIEW 过程中遇到的一些问题，以及 LabVIEW 一些不太完善的方面。

https://lv.qizhen.xyz/appendix_languages

**Comment by ZTGithub01** [comment]
这本书在工作期间看的差不多，希望阮老师能对框架部分增加些介绍，qmh，dqmh，消息队列，这些

**Reply by ruanqizhen** [reply]
我现在工作彻底用不到 LabVIEW 了，只能有空的时候慢慢添加一些。
